.dev/assets/design-styles/trendy/css/blocks/button.scss

```diff
@@ -21,7 +21,7 @@
 		&::after {
 			margin-left: 10px;
 			opacity: 1;
-			width: 10px;
+			width: 20px;
```

Comment:
This appears to work fine up until `--go-button--font-size` is `1.5rem`, then it starts to get cut off again. Perhaps that's fine for typical use cases.